mod-source-record-manager-server/src/main/java/org/folio/dao/JobExecutionFilter.java

```diff
@@ -127,10 +135,17 @@ public String buildCriteria() {
     if (completedBefore != null) {
       addCondition(conditionBuilder, buildLessThanOrEqualCondition(COMPLETED_DATE_FIELD, formatter.format(completedBefore)));
     }
+    setIsDeletedCondition(conditionBuilder);
     addCondition(conditionBuilder, buildNotBoolCondition(JOB_PROFILE_HIDDEN_FIELD));
     return conditionBuilder.toString();
   }
 
+  private void setIsDeletedCondition(StringBuilder conditionBuilder) {
+    if(!isDeleted){
```

Comment:
Or even better to do the same as for hidden, so reuse this method: buildNotBoolCondition(JOB_PROFILE_HIDDEN_FIELD)
Because we don't need to show only deleted, so to simplify can follow approach with is_profile_hidden field

Reply:
Hi @SerhiiNosko - code has been updated as per your suggestions, Thank you 🙂